### ISSUE TITLE
Bug Report: Description of services is missing

### ISSUE THREAD

**Issue by ErikKoppenhagen** [OpenKieler/osgiviz#9]
The model generator doesn't seem to include the javadoc on services/interfaces/classes/... while generating the model. Their description is empty. Additionally, there seems to be a different handling for the description of OSGi services and interfaces:
![OSGiViz Description](https://user-images.githubusercontent.com/60260431/73073793-6bb2b580-3eb8-11ea-975e-3a6e61e3ba83.png)

**Comment by NiklasRentzCAU** [comment]
Yes, I noticed that as well. Once models can be generated after #10, this can be fixed in the model generator bundle.
